### PR TITLE
Add requirement matching

### PR DESCRIPTION
**Pull request by joejulian**

### CODE REVIEW

**Comment by joejulian** [comment]
This requires https://github.com/mesosphere/kubeaddons/pull/277

**Review by shaneutt — DISMISSED** [review]
:+1: just need to resolve the conflicts then :heavy_check_mark:

File: templates/localvolumeprovisioner.yaml [hunk 1]
```diff
@@ -4,6 +4,9 @@ kind: Addon
 metadata:
   name: localvolumeprovisioner
   namespace: kubeaddons
+  labels:
+    kubeaddons.mesosphere.com/provides: storageclass
+    kubeaddons.mesosphere.com/provides: defaultstorageclass
```
Comment (alejandroEsc) [inline templates/localvolumeprovisioner.yaml]: can you have duplicate labels?

File: templates/awsebscsidriverstorageclassdefault.yaml [hunk 2]
```diff
@@ -6,9 +6,13 @@ metadata:
   namespace: kubeaddons
   labels:
     kubeaddons.mesosphere.com/provides: storageclass
+    kubeaddons.mesosphere.com/provides: defaultstorageclass
```
Comment (alejandroEsc) [inline templates/awsebscsidriverstorageclassdefault.yaml]: can you have duplicate labels? Honestly asking here, never seen this before, should this be a list then?

Comment (alejandroEsc) [reply templates/awsebscsidriverstorageclassdefault.yaml]: yeah i looked it upt, should be unique. So i am guessing this was a mistake.

**Review by shaneutt — DISMISSED** [review]
Once @alejandroEsc's comments are :heavy_check_mark:, then :heavy_check_mark:

**Review by shaneutt — APPROVED** [review]

**Pull request merged**